03_letters/joel00wood/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	fmt.Println(strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	out := map[rune]int{}
+	for _, runeValue := range s {
+		out[runeValue]++
+	}
+	return out
+}
+
+func sortLetters(m map[rune]int) []string {
+	out := []string{}
```

Comment:
As you know the length of the array you want, you can allocate the size when defining, has some perf benefits - https://github.com/alexkohler/prealloc

Reply:
That's true, definitely seems worth implementing if not solely to get into the habit of doing so where appropriate

Reply:
Implemented